systemtest/src/test/java/io/strimzi/systemtest/specific/RackAwarenessST.java

```diff
@@ -321,7 +321,7 @@ void testMirrorMaker2RackAwareness(ExtensionContext extensionContext) {
 
     private void consumeDataWithNewSinkConnector(ExtensionContext extensionContext, String newConnectorName, String connectClusterName, String topicName, String namespace) {
 
-        LOGGER.info("Deploy Sink connector in Kafka Connect Cluster: {}/{}", namespace, newConnectorName);
+        LOGGER.info("Deploy Sink connector in KafkaConnect Cluster: {}/{}", namespace, newConnectorName);
```

Comment:
```suggestion
        LOGGER.info("Deploy Sink connector in KafkaConnect cluster: {}/{}", namespace, newConnectorName);
```
+ `sink connector` or `Sink Connector` or `sink KafkaConnector`